test/test_functional_tensor.py

```diff
@@ -68,6 +69,12 @@ def test_adjustments(self):
                 max_diff = (ft_img - f_img).abs().max()
                 self.assertLess(max_diff, 5 / 255 + 1e-5)
 
+    def test_rgb_to_grayscale(self):
+        img_tensor = torch.randint(0, 255, (3, 16, 16), dtype=torch.uint8)
+        grayscale_tensor = F_t.rgb_to_grayscale(img_tensor).to(int)
+        grayscale_pil_img = torch.tensor(np.array(F.to_grayscale(F.to_pil_image(img_tensor)))).to(int)
+        self.assertTrue(torch.allclose(grayscale_tensor, grayscale_pil_img, 1e-1))
```

Comment:
Can you change using `torch.allclose` with something like
```
max_diff = (grayscale_tensor - grayscale_pil_img).abs().max()
self.asertLess(...)
```
so that it gives a better idea of how far off are the two implementations of grayscale?

Reply:
Hi @fmassa , the max diff will be 1 which arises maybe due to rounding.